keylime/ima/ima_dm.py

```diff
@@ -385,7 +400,7 @@ def _check_attr(attr: Optional[Union[int, str, bool]], reference_value: Optional
 
         if isinstance(attr, str):
             try:
-                return reference_value == bool(strtobool(attr))
+                return reference_value == bool(_strtobool(attr))
```

Comment:
Now that we return a bool, we can skip the conversion

Reply:
Yes, I'll fix that.